src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -46,6 +47,15 @@ const FilterConditionSelector: React.FC<Props & WithOutsideClickProps> = ({
 	const searchRef = React.createRef<Input>();
 	const listRef = React.createRef<HTMLUListElement>();
 	const mainRef = React.useRef<HTMLElement | null>(null);
+	const context = React.useContext(FilterWrapperContext);
+	const [scrollState, setScrollState] = React.useState<ScrollState>("full")
```

Comment:
Под такое лучше **enum** заводить
```suggestion
	const [scrollState, setScrollState] = React.useState<ScrollState>(ScrollState.Full)
```